couchbase/datadog_checks/couchbase/couchbase.py

```diff
@@ -183,6 +188,8 @@ def check(self, _):
         self._create_metrics(data)
         if self._sync_gateway_url:
             self._collect_sync_gateway_metrics()
+        if self._index_stats_url:
```

Comment:
Should this also check that the Couchbase version is at least 7.0?

Reply:
I thought about it, I can definitely do it in the _collect_version() function and add a return for major version, but I was thinking about the edge case in which for whatever reason the version collection fails, we won't collect the index stats even if it's available. I didn't encounter this issue before though and could be a moot case. What do you think?

Reply:
Actually it might not be a concern since if the metadata can't be collected, the check will not run anyways. I'll go ahead and try adding the logic to check for version 7+.